### ISSUE TITLE
Focus: Shock should charge creepers again

### ISSUE THREAD

**Issue by MalkContent** [Azanor/thaumcraft-suggestions#4]
I almost consider this not being a thing anymore being a bug, but I didn't wanna get cheeky with it now that we have this fine repository here :]

**Comment by Azanor** [comment]
hehe

**Comment by MalkContent** [comment]
From a balancing point: the shock focus is arguably the most powerful focus for regular combat, 2/3 hitting your standard variety vanilla mobs at up to - I'd call it - medium-long range with no travel time or ballistic shooting involved. Charged creepers however are deliciously dangerous to all but the most op-modded players. So at least "the best" focus would gain back it's downside :)
Suggesting chance to charge to scale with potency. Static field (the aoe thingy, don't remember) should always charge them.

**Comment by carso150** [comment]
That is true, the shock foci is op, too op, why I need to use another tipe of foci if I have a shock lv 3, it need rebalacing

**Comment by MalkContent** [comment]
Meh. Not like it oneshots everything. It's just that for regular use there's no equal alternative. Frost focus is difficult to use and does about the same damage. The optional slow effect on it isn't really worth it if you just want to kill a zombie, imo. (maybe make it stack with multiple hits and/or reduce armor). And fire focus doesn't penetrate anymore, so the "incinerate large groups of mobs" upside kind of vanished a bit.  
Hellbats are cool but slow and overkill for your average zombie. Nice for guardians or champions.  
Fireball still is the king though :D  
I forgot where I was going with this.

**Comment by carso150** [comment]
That the shock foci is not that op
